hoomd/mpcd/CellList.cc

```diff
@@ -517,51 +550,136 @@ void mpcd::CellList::buildCellList()
     m_conditions.resetFlags(conditions);
     }
 
-/*!
- * \param timestep Timestep that the sorting occurred
- * \param order Mapping of sorted particle indexes onto old particle indexes
- * \param rorder Mapping of old particle indexes onto sorted particle indexes
- */
-void mpcd::CellList::sort(uint64_t timestep,
-                          const GPUArray<unsigned int>& order,
-                          const GPUArray<unsigned int>& rorder)
+void mpcd::CellList::finishComputeProperties()
     {
-    // no need to do any sorting if we can still be called at the current timestep
-    if (peekCompute(timestep))
-        return;
-
-    // if mapping is not valid, signal that we need to force a recompute next time
-    // that the cell list is needed. We don't call forceCompute() directly because this
-    // always runs compute(), and we just want to defer to the next compute() call.
-    if (rorder.isNull())
+    if (!m_property_sum)
         {
-        m_force_compute = true;
         return;
         }
```

Comment:
I see this is the one spot where this flag gets checked. If `finishComputeProperties()` is only invoked once as part of the compute method, then I don't think it is actually needed. This can probably be removed.